### ISSUE TITLE
Set `last-call-deadline` in the preamble before merging relevant pull requests

### ISSUE THREAD

**Issue by SamWilsn** [ethereum/EIP-Bot#66]
_Originally from #55_

Immediately before merging (IMO, ideally in the merge commit itself) set `last-call-deadline` in the preamble to 14 days after the merge date.

**Comment by JEAlfonsoP** [comment]
I will work on it.

**Comment by MicahZoltu** [comment]
I'm weakly against this one because last call is 14 days minimum, but unlimited maximum.  Some authors like to give a longer last call, and core EIPs have a last call that runs until mainnet launch generally.

**Comment by MicahZoltu** [comment]
Perhaps, if last call is missing automatically add it with 14 day window, if it is present then do nothing?

**Comment by SamWilsn** [comment]
How about:

 * **If the last call deadline is missing:** set it to merge date plus 14 days.
 * **If the last call deadline is present, but less than 14 days from the merge date:** set it to merge date plus 14 days.
 * **If the last call deadline is present, and is at least 14 days from the merge date:** leave untouched.

**Comment by JEAlfonsoP** [comment]
could you please review this issue against 53. ? I believe 53 covers both, time and editors approval. 
If this is the case I will make a PR following Issue 53 to cover both.

**Comment by MicahZoltu** [comment]
#53 is different.  This suggestion is about making the bot add/adjust the last call deadline header field.  #53 is about changing the number of editors required to change EIP-1.

**Comment by JEAlfonsoP** [comment]
Ok. Roger that.

where would you like last-call-deadline to be added (or where normally is added, I do not see it in the EIP-1 template):

---
eip: <to be assigned>
title: <The EIP title is a few words, not a complete sentence>
description: <Description is one full (short) sentence>
author: <a comma separated list of the author's or authors' name + GitHub username (in parenthesis), or name and email (in angle brackets).  
discussions-to: <URL>
status: Draft
type: <Standards Track, Meta, or Informational>
category (*only required for Standards Track): <Core, Networking, Interface, or ERC>
created: <date created on, in ISO 8601 (yyyy-mm-dd) format>
requires (*optional): <EIP number(s)>
---

**Comment by MicahZoltu** [comment]
After `status` line.
```
...
status: Last Call
last-call-deadline: YYYY-MM-DD
type: Core
...
```

**Comment by JEAlfonsoP** [comment]
is anybody working on it ?

**Comment by Pandapip1** [comment]
Nope, go ahead! This might end up being a separate project though, so don't spend too much time on it.

**Comment by JEAlfonsoP** [comment]
Ok.

**Comment by JEAlfonsoP** [comment]
> How about:
> 
> * If the last call deadline is **missing:** set it to merge date plus 14 days.
> * If the last call deadline is **present**, but **sooner** than 14 days plus the merge date: set it to merge date plus 14 days.
> * If the last call deadline is **present**, and is **later** than 14 days plus the merge date: leave untouched.

Question: Should EIP-Bot follows the above  If's ?

**Comment by Pandapip1** [comment]
Yes, but see my comment https://github.com/ethereum/EIP-Bot/issues/65#issuecomment-1237530860

**Comment by JEAlfonsoP** [comment]
Roger. I will pass on this one either. Ty.
